db/migration/src/m20240226_000002_create_ticket.rs

```diff
@@ -0,0 +1,73 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Ticket::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Ticket::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(Ticket::ChannelId).string_len(64).not_null())
+                    .col(ColumnDef::new(Ticket::Amount).string_len(50).not_null())
```

Comment:
better use `binary12` here. string conversion can be lossy and u96 doesn't fit into SQlite integers